front-end/src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,24 @@
+import { Logo } from "../../components/logo"
+import { Card } from "../../style-guide/Card"
+import { Input } from "../../style-guide/Input";
+import { SubTitle } from "../../style-guide/SubTitle"
+import { Title } from "../../style-guide/Title"
```

Comment:
We can try to setup absolute imports, so we don't ahve these many `../../../../`